subjects/mobile-dev/bouncer/README.md

```diff
@@ -1,27 +1,35 @@
-# Bouncer
+## Bouncer
 
-### Introduction
+The main objective of this project is to demonstrate how to work with phone hardware, specifically the [accelerometer](https://www.credencys.com/blog/accelerometer/#:~:text=What%20does%20an%20accelerometer%20in,the%20orientation%20of%20your%20app.). By the end, you will have a fully functional mobile game that can be played on your phone.
 
-Some apps require using phone's hardware. In this app you are presented such an opportunity.
+### Instructions
 
-Develop an app that uses [accelerometer](https://www.credencys.com/blog/accelerometer/#:~:text=What%20does%20an%20accelerometer%20in,the%20orientation%20of%20your%20app.). The app should be a smaller version of the 90s game of bouncing ball, where ball would be moving around, and player’s objective is to prevent it from touching the bottom part of the screen.
+The app should be a simplified version of the classic `90s bouncing ball game`, where the player's objective is to destroy all `blocks` in the screen and prevent the `ball` from touching the bottom of the screen as it moves around, for that you will have to use the phone's `accelerometer` to control the `player's tile`.
 
 ### Rules of the game
 
-- Ball moves in a linear manner
-- Ball changes its direction when it hits wall or tiles. The direction change happens by the [rule](https://www.toppr.com/ask/en-af/question/1865186/): the degree it came with should be degree it goes out, similar to real life experience.
-- All tiles except player's tile disappear when they are hit with a ball
-- Game ends when ball touches bottom part of the phone or all the tiles are disappeared.
-- Player's tile moves according to phone's tiltiness. If phone is tilted more towards right side, so does the tile, the same with left side.
+- The `ball` moves in a linear manner and changes direction when it hits a `wall` a `block` or the `player's tile`. The direction change happens by the [rule](https://www.toppr.com/ask/en-af/question/1865186/): the degree it came with should be the degree it goes out, similar to real-life experience.
+- All `blocks`, disappear when they are hit by the `ball`.
```

Comment:
Here I think it will be better if we remove the `,`

Reply:
done